jenkins/setup.sh

```diff
@@ -26,11 +26,11 @@ fi
 
 logdir=/tmp/gridtools/ # log to subfolder to workaround https://webrt.cscs.ch/Ticket/Display.html?id=38406
 mkdir -p $logdir
-chmod +rwx $logdir
+chmod ugo+rwx $logdir
```

Comment:
should we fall over to /tmp if fail? or exit?

Reply:
after the settings are ok in all nodes, this should never fail?

Reply:
Alternative to this approach could be a $USER suffix in the log dir.

Reply:
I would vote for the `$USER` prefix.
We probably should use `/var/tmp/` to be able to read logs after the node reboots.

Reply:
I'll use a suffix to be make it easier to find all `gridtools_` folders.